src/components/Front/Pages/Landing/CallToAction.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+class CallToAction extends Component {
+    state = {}
+    render() {
+        return (
+            <section id="try" class="section-spacer text-left pb-0 pt-50">
```

Comment:
@DanielBaker13 This will leave an error.  in React you will have to use className instead of class. it will not transpile it properly.